src/elements/core/mixins/named-slot-list-mixin.ts

```diff
@@ -32,12 +37,17 @@ export declare abstract class SbbNamedSlotListMixinType<
 > extends SbbHydrationMixinType {
   protected abstract readonly listChildLocalNames: string[];
   @state() protected listChildren: C[];
-  protected renderList(attributes?: {
-    class?: string;
-    ariaLabel?: string;
-    ariaLabelledby?: string;
-  }): TemplateResult;
-  protected listSlotNames(): string[];
+  protected renderList(
+    attributes?: {
+      class?: string;
+      ariaLabel?: string;
+      ariaLabelledby?: string;
+    },
+    listItemAttributes?: {
+      localNameVisualOnly?: string[];
+    },
+  ): TemplateResult;
+  protected listSlot(): SbbNamedSlotProperties[];
```

Comment:
should it maybe be slotList? or listSlots? or listSlotEntries? Now it sounds weird, as it is a list but sounds like a single slot